lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -10706,6 +10768,28 @@ namespace Js
         return requestContext->GetMissingPropertyResult();
     }
 
+    Var JavascriptOperators::GetPropertyNoCache(RecyclableObject* instance, PropertyId propertyId, ScriptContext* requestContext)
+    {
+        return JavascriptOperators::GetPropertyNoCache(instance, instance, propertyId, requestContext);
+    }
+
+    Var JavascriptOperators::GetPropertyNoCache(Var instance, RecyclableObject* propertyObject, PropertyId propertyId, ScriptContext* requestContext)
+    {
+        Var value;
```

Comment:
nit: I see that the current implementation of GetProperty_InternalSimple definitely sets value; however, could you initialize it to nullptr here?